rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1.java

```diff
@@ -61,15 +61,33 @@
 			headers = {"Accept=" + Constants.V1_API_ACCEPT})
 	public ResponseEntity<InputStreamResource> getFileById(@PathVariable("fileId") String fileId)
 			throws IOException {
-		API_LOG.info("CPC+ file request received");
+		API_LOG.info("CPC+ file retrieval request received");
 
 		InputStreamResource content = cpcFileService.getFileById(fileId);
 
-		API_LOG.info("CPC+ file request succeeded");
+		API_LOG.info("CPC+ file retrieval request succeeded");
 
 		HttpHeaders httpHeaders = new HttpHeaders();
 		httpHeaders.setContentType(MediaType.APPLICATION_XML);
 
 		return new ResponseEntity<>(content, httpHeaders, HttpStatus.OK);
 	}
+
+	/**
+	 * Updates a file's status to processed in the database
+	 *
+	 * @param fileId Identifier of the file needing to be updated
+	 * @return Message if the file was updated or not
+	 */
+	@RequestMapping(method = RequestMethod.POST, value = "/file/{fileId}",
+			headers = {"Accept=" + Constants.V1_API_ACCEPT} )
+	public ResponseEntity<String> markFileProcessed(@PathVariable("fileId") String fileId) {
```

Comment:
I just merged in the feature flag PR https://github.com/CMSgov/qpp-conversion-tool/pull/518
Can you please make sure to add a similar check to this method (and a test) too?

Reply:
Done and done